Analyses/src/CaloExample_module.cc

```diff
@@ -82,20 +82,20 @@ namespace mu2e {
 
        TH1F *hcryE_,*hcryT_,*hcryX_,*hcryY_,*hcryZ_;
        TH1F *hcluE_,*hcluT_,*hcluX_,*hcluY_,*hcluZ_,*hcluE_1Et,*hcluE_1E9,*hcluE_1E25,*hcluE_F;       
-       TH2F *hxy_;
+       TH2F *hxy_,*hECE;
 
        TTree* Ntup_;
        int   _evt,_run;
 
        int   nGen_,genPdgId_[16384],genCrCode_[16384];
        float genmomX_[16384],genmomY_[16384],genmomZ_[16384],genStartX_[16384],genStartY_[16384],genStartZ_[16384],genStartT_[16384];
 
-       int   nHits_,cryId_[163840],crySectionId_[163840],crySimIdx_[163840],crySimLen_[163840];
-       float cryEtot_,cryTime_[163840],cryEdep_[163840],cryPosX_[163840],cryPosY_[163840],cryPosZ_[163840],_cryLeak[163840];
+       int   nHits_,cryId_[16384],crySectionId_[16384],crySimIdx_[16384],crySimLen_[16384];
```

Comment:
What is this value?  It should be a parameter if it can change, or a constexpr if not.

Reply:
Done